modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/util/HTTPUtil.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.gradle.plugins.workspace.internal.util;
+
+import com.liferay.petra.string.CharPool;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * @author Thiago Buarque
+ */
+public class HTTPUtil {
+
+	public static boolean isUrl(String url) {
+		return isUrl(url, false);
+	}
+
+	public static boolean isUrl(String url, boolean acceptRootRelative) {
+		if (!StringUtil.isBlank(url)) {
+			if (acceptRootRelative && (url.charAt(0) == '/')) {
+				return true;
+			}
+
+			if (url.indexOf(CharPool.COLON) == -1) {
+				return false;
+			}
+
+			try {
+				new URL(url);
+
+				return true;
+			}
+			catch (MalformedURLException malformedURLException) {
+			}
+		}
+
+		return false;
+	}
```

Comment:
I think we can just put this into StringUtil.
Also, I don't think there's ever a case in this context where we'd want to `acceptRootRelative`, right? If not then we can remove the overload and remove the short-circuit.

Reply:
Done, waiting current CI run to finish to make sure nothing in the logic is wrong